component/src/main/java/com/celements/blog/metatag/HeaderSocialMediaMetaTags.java

```diff
@@ -0,0 +1,58 @@
+package com.celements.blog.metatag;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xwiki.component.annotation.Component;
+import org.xwiki.component.annotation.Requirement;
+
+import com.celements.blog.article.Article;
+import com.celements.blog.plugin.EmptyArticleException;
+import com.celements.metatag.MetaTag;
+import com.celements.metatag.MetaTagProviderRole;
+import com.celements.model.context.ModelContext;
+import com.celements.pagetype.PageTypeReference;
+import com.celements.pagetype.service.IPageTypeResolverRole;
+import com.xpn.xwiki.XWikiException;
+
+@Component(HeaderSocialMediaMetaTags.COMPONENT_NAME)
+public class HeaderSocialMediaMetaTags implements MetaTagProviderRole {
```

Comment:
Added @NotThreadSave since I'm not sure if it is (I suppose not: has several Maps fields) -> lecture / more info about your (and @msladek) thread safety research from last year needed.